app/reducers/index.ts

```diff
@@ -1,10 +1,13 @@
 import { combineReducers } from "redux";
+import { reducer as form } from "redux-form";
 import icoState, { IcoState } from "./icoState";
 
 export interface IAppState {
   readonly icoState: IcoState;
+  readonly form: any;
```

Comment:
You can get state shape from `redux-form` library - it's `FormState`

Reply:
done